test/suites/route-response-duplicate.spec.ts

```diff
@@ -0,0 +1,134 @@
+import { Tests } from 'test/lib/tests';
+import { expect } from 'chai';
+
+const tests = new Tests('basic-data');
+
+describe.only('Duplicate a route response', async () => {
```

Comment:
Please remove the forgotten `.only`. And the `async`. Describes shouldn't be async.

Also, I think you could definitely "regroup" the tests. Duplicate once, in a first `it`, and check the headers, body, rules, etc in subsequent `it`. I usually have dependent `it` test cases, for each steps of the test, and use different `describe` to separate each test cases (with a call to `new Tests()` inside each describe to start with a fresh app/state). You can still keep it like this if you want. 👍

Reply:
I suppose I'll go with this approach of grouping. Thanks for advice though.